ui/components/requirements/requirement.jsx

```diff
@@ -15,13 +16,37 @@ export default function Requirement({ requirement }) {
       <div className="req-text col col-10">
         { reqTexts }
         <div className="clearfix mt3">
-          <div className="applies-to mr2">
+          <div className="applies-to mr2 metadata">
             Applies to: [not implemented]
           </div>
-          <div className="sunset-date">
-            Sunset date by { requirement.policy.sunset || 'none' }
+          { requirement.issuing_body ? (
+            <div className="issuing-body metadata">
+              Issuing body: { requirement.issuing_body }
+            </div>
+          ) : (
+            ''
+          ) }
```

Comment:
Generally, we don't want this kind of logic in the template as it makes them harder to read. Mind moving this into a constant instead? Also, `null` is the more generally way to say "nothing here".